extras/create_send_notifications.py

```diff
@@ -55,10 +55,11 @@ def build_notifications_org_list(db):
         # If the notification document's ticket owner has "CYHY" in their list of report_types,
         # then a notification should be generated for that owner:
         notifications_to_generate.add(request["_id"])
+        logging.debug("{} - Log request id".format(request["_id"]))
         # Recursively check for any parents of the ticket owner that have "CYHY" in
         # their list of report_types.  If found, add them to the list of owners that
         # should get a notification.
-        cyhy_parent_ids = cyhy_parent_ids | find_cyhy_parents(db, request["_id"])
+        cyhy_parent_ids.update(find_cyhy_parents(db, request["_id"]))
```

Comment:
Why is this part of this commit?  The commit message does not mention anything that has anything to do with this particular change.  This was previously in a separate commit, was it not?

Reply:
Sorry 😅

Reply:
You need to fix this before I will approve.  I think @mcdonnnj needs to be listed as a co-author on the commit you separate out that contains this change.

Reply:
@jsf9k Please see https://github.com/cisagov/cyhy-reports/pull/96/commits/9e4dcc1ae51e83a30b2a14d084f25069c9acf37e

Reply:
Thank you!